### DISCUSSION TITLE
[Feature Request]: add graphql support

### DISCUSSION THREAD

**Discussion by wfpaisa** [strapi-community/plugin-io#26] [Ideas]
### What issue are you experiencing?

Get all events when I use Graphql

  "dependencies": {
    "@strapi/plugin-documentation": "^4.1.7",
    "@strapi/plugin-graphql": "^4.1.7",
    "@strapi/plugin-i18n": "4.1.7",
    "@strapi/plugin-users-permissions": "4.1.7",
    "@strapi/provider-email-nodemailer": "^4.1.8",
    "@strapi/strapi": "4.1.7",
    "better-sqlite3": "7.4.6",
    "strapi-plugin-io": "^1.1.0"
  },

### Steps To Reproduce

Add Graphql plugin.  :  "@strapi/plugin-graphql": "^4.1.7",

### What version of the plugin are you using?

 "strapi-plugin-io": "^1.1.0"

### What strapi version are you using?

 "@strapi/strapi": "4.1.7",

**Comment by ComfortablyCoding** [comment]
This is not a bug, currently graphql is not supported.

**Comment by wfpaisa** [comment]
oh, thanks for your answer, this functionality would be great

**Reply by lionhu** [reply]
You can use middleware to fulfill the support of GraphQL

**Reply by Avi-ADAM** [reply]
i be happy  to know more about this option

**Comment by ComfortablyCoding** [comment]
Graphql is now natively supported in v2 of this plugin

**Reply by Avi-ADAM** [reply]
thank you for that. is there a way for modifications for the data passelient , to add population or with graphql to specify what data should be passed?

**Reply by ComfortablyCoding** [reply]
At the moment no, tracking that via #82 as it's a similar use case.
